### PR TITLE
[iOS] - Fix AIChat feedback options and chat intro layout

### PR DESCRIPTION
**Pull request by Brandon-T**
## Summary

- Adjust string to have a period/dot at the end.
- Fix feedback options to be non-localized when sent to the server.
- Update layout of Chat Intro to be vertically stacked.
- Update page-context view to be always light-mode colour.

Resolves https://github.com/brave/brave-browser/issues/36870 and https://github.com/brave/brave-browser/issues/36135

## Submitter Checklist:

- [x] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackView.swift [hunk 1]
```diff
@@ -309,17 +309,31 @@ private struct AIChatFeedbackLeoPremiumAdView: View {
...
+    case .notHelpful:
+      return "not-helpful"
+    case .incorrect:
+      return "incorrect"
+    case .unsafeHarmful:
+      return "unsafe-harmful"
+    case .other:
+      return "other"
+    }
+  }
```
Comment (kylehickinson) [inline ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackView.swift:327]: This change doesn't seem needed

Comment (Brandon-T) [reply ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackView.swift:327]: It is. I need to send exactly just the `id` to the server. The server is seeing localized strings when the user selects an option. But I want the server to see just these strings that aren't localized.

Comment (Brandon-T) [reply ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackView.swift]: Added the comment. Will address the struct vs. enum in the follow up PR I'm opening.

**Review by kylehickinson — APPROVED** [review]

**Comment by kjozwiak** [comment]
Verification **PASSED** on `iPhone 11` running `iOS 17.4.1` using the following build(s):

```
1.67.59 Chromium: 124.0.6367.118 (Official Build) nightly (64-bit) 
--- | ---
Revision | 96b3194901e5...
OS | iOS
```

### `Test Case #1` - https://github.com/brave/brave-browser/issues/36870

Using the STR/Cases mentioned via https://github.com/brave/brave-browser/issues/36870#issue-2190684112, ensured that the `Leo` icon is being stacked rather than appearing besides the results/summaries as per the following:

**`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------
IMG_0240 | IMG_0241 | IMG_0242

As per https://github.com/brave/brave-core/pull/23129#issue-2248711131 via `Update page-context view to be always light-mode colour.`, ensured that the switch is always using `purple` in both `Light` & `Dark` modes as per the following:

**`Example`** | **`Example`**
----------------|---------------
IMG_0245 | IMG_0244

### `Test Case #2` - https://github.com/brave/brave-browser/issues/38022

Using the STR/Cases outlined via https://github.com/brave/brave-browser/issues/38022#issue-2276336812, ensured that `Explore different AI models` descriptive has a `.` at the end of the sentence as per the following:

IMG_0243

**Pull request merged**